src/content/docs/build/set-up-options/self-serve-portal-for-orgs.mdx

```diff
@@ -0,0 +1,92 @@
+---
+page_id: a2668524-5842-4c68-ab50-30b7e8c3e842
+title: Enable self-serve portal for orgs
+sidebar:
+  order: 7
+relatedArticles:
+  - beebe908-27f8-4be6-b74c-05fc8ff5dfb5
+---
+
+You can configure a self-serve portal to enable authorized organization members to be able to self-manage functions provided by Kinde. For example, you can allow them to update their business details, payment details (if you have billing set up), multi-factor auth settings, and manage organization members.
```

Comment:
@clairekinde11 this portal feature isn't live yet `multi-factor auth settings` and `manage organization members` is enabled only for Kinde instance